plugins/tracers/gstdot.c

```diff
@@ -70,8 +74,10 @@ gst_dot_pipeline_to_file (GstBin * bin, GstDebugGraphDetails flags)
         full_trace_dir);
   }
 
-  full_file_name = g_strdup_printf ("%s" G_DIR_SEPARATOR_S "pipeline.dot",
-      full_trace_dir);
+  strftime (time_suffix, MAX_SUFFIX_LEN, "%F_%T", localtime (&now));
```

Comment:
Why was `strftime` used instead of Glib function `g_date_strftime`?

Reply:
I didn't find an equivalent. According to `g_date_strftime` documentation:

```
Works just like the platform's C library strftime() function, but only accepts date-related formats; time-related formats give undefined results. 
```